crates/fflonk/Cargo.toml

```diff
@@ -0,0 +1,18 @@
+[package]
+name = "fflonk"
+version = "0.1.0"
+edition = "2021"
```

Comment:
Could you add `license = "MIT OR Apache-2.0"` so the crate does not appear to be unlicensed?